Document management interface DHCP configuration file (#122)
bisdn/bisdn.github.io: getting_started/basic_networking.md

It is possible to configure the loopback interface with other IPv4 and IPv6 addresses, thus providing connectivity to the loopback interface itself. In order to reach this interface, a "via" route must be present.

## Management interface

> The BISDN Linux switches have a management interface, configured via `systemd-networkd`. By default, this file configures the interface to use DHCP, and can be opened as follows.

Reviewer: ```suggestion
BISDN Linux uses `systemd-networks` to configure the management interface, which by default is set to use DHCP. You can edit the configuration file shown below to match your prefered setup. Please see the [systemd-networkd docs](https://www.freedesktop.org/software/systemd/man/systemd.network.html) for more information.
```

```

> root@bisdn-linux# cat /lib/systemd/network/80-wired.network

Reviewer: ```suggestion
/lib/systemd/network/80-wired.network
```

Reviewer: Also suggest to move to the `filename` above the code block, and then the contents inside the codeblock as we do in the bgp network configuration: https://github.com/bisdn/bisdn.github.io/edit/master/network_configuration/bgp.md line 72

> BISDN Linux uses `systemd-networks` to configure the management interface, which by default is set to use DHCP. You can edit the configuration file shown below to match your prefered setup. Please see the [systemd-networkd docs](https://www.freedesktop.org/software/systemd/man/systemd.network.html) for more information.

Reviewer 2: ```suggestion
BISDN Linux uses `systemd-networkd` to configure the management interface, which by default is set to use DHCP. You can edit the configuration file shown below to match your prefered setup. Please see the [systemd-networkd docs](https://www.freedesktop.org/software/systemd/man/systemd.network.html) for more information.
```

AFAIK Files in /lib aren't supposed to be edited by the user, you are supposed to create your own in /etc/systemd./network/

Reviewer 2: Especially since 80-wired.network is *not* marked as a configuration file, so it won't be backed up on update.

> BISDN Linux uses `systemd-networks` to configure the management interface, which by default is set to use DHCP. You can edit the configuration file shown below to match your prefered setup. Please see the [systemd-networkd docs](https://www.freedesktop.org/software/systemd/man/systemd.network.html) for more information.

Reviewer 3: i agree with @KanjiMonster here, i think we should not encourage people to edit this file, but rather document that you can just add another file to /etc/systemd/network to overwrite this file (e.g. by choosing a lower number for the file name, so it will be read first and the 80-wired filed will be ignored)

> BISDN Linux uses `systemd-networkd` to configure the management interface, which by default is set to use DHCP. To overwrite the configuration of this file, add a custom file on the `/etc/systemd/network` folder, with a prefix number lower than `80`. Please see the [systemd-networkd docs](https://www.freedesktop.org/software/systemd/man/systemd.network.html) for more information.

Reviewer: ```suggestion
BISDN Linux uses `systemd-networkd` to configure the management interface, which by default is set to use DHCP. To overwrite the configuration of this file, add a custom file in the `/etc/systemd/network` directory, with a prefix number lower than `80`. Please see the [systemd-networkd docs](https://www.freedesktop.org/software/systemd/man/systemd.network.html) for more information.
```

> BISDN Linux uses `systemd-networkd` to configure the management interface, which by default is set to use DHCP. To overwrite the configuration of this file, add a custom file in the `/etc/systemd/network` directory, with a prefix number lower than `80`. Please see the [systemd-networkd docs](https://www.freedesktop.org/software/systemd/man/systemd.network.html) for more information.

Reviewer 3: systemd-networkd is not by default set to use DHCP

> BISDN Linux uses `systemd-networkd` to configure the management interface, which by default is set to use DHCP. To overwrite the configuration of this file, add a custom file in the `/etc/systemd/network` directory, with a prefix number lower than `80`. Please see the [systemd-networkd docs](https://www.freedesktop.org/software/systemd/man/systemd.network.html) for more information.

Reviewer 3: please explain why lower than 80  and what "this file" is

> BISDN Linux uses `systemd-networkd` to configure the management interface, which by default is set to use DHCP. To overwrite the configuration of this file, add a custom file in the `/etc/systemd/network` directory, with a prefix number lower than `80`. Please see the [systemd-networkd docs](https://www.freedesktop.org/software/systemd/man/systemd.network.html) for more information.

Reviewer 3: ```suggestion
BISDN Linux uses `systemd-networkd` to configure the management interface. To allow users to ssh to and configure the switch without any further configuration after installing an image, our yocto build chain adds a default configuration file to `/lib/systemd/network/80-wired.network` (shown below), which configures all interfaces named "eth*" or "en*" to use DHCP. By using this configuration, the management interface of all switch platforms automatically uses DHCP for its own address configuration. To overwrite this configuration file, you can add a custom file in the `/etc/systemd/network` directory, with a prefix number lower than `80`. The lower prefix will make sure this file is read first when starting systemd-networkd (only the first `Match` for each interface is applied). Please see the [systemd-networkd docs](https://www.freedesktop.org/software/systemd/man/systemd.network.html) for more information.
```
